<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>换肤中心</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="SkinListWidth" name="listWidget_Skin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
  </widget>
  <widget class="CBaseTitleBar" name="widget_change_Title" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QToolButton" name="toolButton_Close">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>23</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/back_group/Resources/min_button/minback.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/min_button/close.png</normaloff>Resources/min_button/close.png</iconset>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonFollowStyle</enum>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_Min">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>23</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/back_group/Resources/min_button/minback.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/min_button/minb1.png</normaloff>Resources/min_button/minb1.png</iconset>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonFollowStyle</enum>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/back_group/Resources/min_button/catalog_8.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SkinListWidth</class>
   <extends>QListWidget</extends>
   <header>skinlistwidth.h</header>
  </customwidget>
  <customwidget>
   <class>CBaseTitleBar</class>
   <extends>QWidget</extends>
   <header>../Common/CommonUI/CommonTitleBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="saftui.qrc"/>
 </resources>
 <connections/>
</ui>
